src/lib/gas.ts

```diff
@@ -151,3 +150,31 @@ export const createGasPrice = async (
 
   return insertedEnrichedGas;
 };
+
+export const getEnrichedGases = async (
+  trx?: Knex.Transaction
+): Promise<EnrichedGas[]> => {
+  const transaction = trx ?? knexController;
```

Comment:
```suggestion
export const getEnrichedGases = async (): Promise<EnrichedGas[]> => {
```

Tätä ei ainakaan tällä hetkellä käytetä missään niin että tänne annettaisiin transaktio => voitaisiin ehkä optimoida yksinkertaisuuden ehdoilla ja käyttää suoraan tuota knexControlleria?

Toisekseen, tämä on yksinkertainen lukuoperaatio, joka tuskin tarvitsee mitään transaktiota.

Reply:
En nää hyötyä tässä optimoinnissa. Transaktion tarjoaminen mahdollisuutena ei myöskään mielestäni heikennä käytettävyyttä, se vain tarjoaa mahdollisuuksia tulevaisuudessa.

Reply:
Tulevaisuuden varalle (ei tarpeeseen, vaan varalle) ominaisuuksien tekeminen on kyllä aika turhaa.

Reply:
Luettavuus on heihekentynyt